cmd/swarmctl/service/flagparser/flags.go

```diff
@@ -32,6 +32,7 @@ func AddServiceFlags(flags *pflag.FlagSet) {
 	flags.String("memory-limit", "", "memory limit (e.g. 512m)")
 	flags.String("cpu-reservation", "", "number of CPU cores reserved (e.g. 0.5)")
 	flags.String("cpu-limit", "", "CPU cores limit (e.g. 0.5)")
+	flags.String(ThirdPartyResourceFlag, "", "user defined resources request (e.g. gpu=3;fpga=1)")
```

Comment:
I noticed that this flag has slightly different semantics than the one under the same name which defines a node's avaliable resources. Would it make sense to differentiate the flag names?